### DISCUSSION TITLE
App Mobile

### DISCUSSION THREAD

**Discussion by LuisCarlosDev** [diego3g/rsxp-2023#36] [General]
Alguém já iniciou a criação do app?

**Comment by carlosdavid0** [comment]
Também gostaria de saber.

**Comment by magnokf** [comment]
eu,  mas estou cheio de dúvidas. Por isso sou a favor de um canal exclusivo para quem pretende ajudar, colaborar e aprender.

**Comment by LuisCarlosDev** [comment]
@magnokf está fazendo qual tela?

**Reply by magnokf** [reply]
screens/Home/index

**Reply by magnokf** [reply]
Acho q te vi lá no figma..kkkk

**Reply by carlosdavid0** [reply]
Acho que deveríamos criar as issues para sinalizar em qual tela estamos.

**Reply by LuisCarlosDev** [reply]
Estou por lá!

**Reply by magnokf** [reply]
me add lá no Discord magnokf#5713

**Reply by carlosdavid0** [reply]
Cria a issues

**Reply by magnokf** [reply]
me dá um alou la no discord. magnokf#5713

**Comment by diego3g** [comment]
Galera, vou criar ainda hoje todas issues relacionadas à construção do app e do back-end e vocês podem sinalizar em quais estão trabalhando :)

**Comment by LuisCarlosDev** [comment]
@diego3g qual biblioteca de ícones vai ser utilizada?

**Comment by amesq01** [comment]
Bom dia, tentei rodar o app, so que ta dando o erro Invalid Clerk Publishable Key, poderiam me ajudar nisso. Sou iniciante :D

**Reply by Rodrigo001-dev** [reply]
Tu adicionou as suas chaves do Clerk no .env?

**Reply by amesq01** [reply]
adicionei no arquivo .env da pasta mobile, somente a chave do expo, so tem essa chave?

**Reply by amesq01** [reply]
tambem coloquei a secret no arquivo env da pasta server e nao vai

**Reply by amesq01** [reply]
![image](https://user-images.githubusercontent.com/69029706/232230429-71dfdc9a-8aa6-450a-a1e1-2feb7e572b71.png)

so da isso ai

**Reply by Rodrigo001-dev** [reply]
Opa @amesq01, descula a demora para responder, cara eu tava tendo um erro com o Clerk em relação as variáveis de ambiente, estava assim importanto de '@env'

![Screenshot from 2023-04-20 23-12-45](https://user-images.githubusercontent.com/59906558/233524530-e01af4eb-190b-4916-9057-8254202ea0b5.png)

dessa forma apareceia um erro para mim
ai eu passei a variavel com process.env.CLERK_PUBLISHABLE_KEY

![Screenshot from 2023-04-20 23-14-23](https://user-images.githubusercontent.com/59906558/233524708-b70bbbc2-fc9c-45f3-9fdf-7372136831a8.png)

ai funcionou. Talvez resolva

Depois de tu criar um projeto la no clerk tu só precisa adicionar essa variavel no env

![Screenshot from 2023-04-20 23-17-15](https://user-images.githubusercontent.com/59906558/233525121-00c54f30-bd39-4dad-b22e-17f6449c0027.png)

**Reply by amesq01** [reply]
vou tentar, valeu

**Reply by amesq01** [reply]
![image](https://user-images.githubusercontent.com/69029706/233529169-4b605513-eacc-4616-9844-b59df37a1bee.png)

Rodou!! valeu. será pq nao tava pegando importando o env daquela forma?

**Reply by Rodrigo001-dev** [reply]
provavelmente é isso, creio que seja alguma coisa relacionado ao nosso ambiente, então por isso, quando for fazer a PR discarta essa alteração.
